src/app/globals.css

```
@@ -132,37 +167,125 @@
   }
   body {
     @apply bg-background text-foreground;
+    font-size: 14px;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)